index.html

```diff
@@ -3950,42 +3950,197 @@ <h2>Syntaxes</h2>
       <section>
         <h3>JSON-LD</h3>
 
-        <p>
+        <section class="informative">
+          <h4>Data Model Foundations</h4>
+          <p>
+  The world of credentials is vast and varied. A physical wallet places
+  relatively few constraints on the shape and size of those credentials — and it
+  places no restriction on their contents. Likewise, the Verifiable Credential Data
+  Model's use of [[!JSON-LD]] provides the consistent shape for all compliant
+  Verifiable Credentials, so that digital wallets can store credentials with
```

Comment:
I do not think we should use the term 'digital wallet' at all, it's confusing. I like the term agent. I don't think we'll agree on a terminology so I'd omit it entirely.

Reply:
> The phrase "digital wallet" is already in use within the VCDM spec.

I believe this is a mistake we should not double down on.

> Also, the comparison is between "physical wallet" and "digital wallet" so introducing a non-parallel term would add more confusion than clarity.

I believe it's a false comparison, which is why it's dangerous. A digital wallet is much more feature rich, and consequently complex, with many more considerations in all directions. So I agree with @Sakurann 's comment below to remove the text entirely.

Reply:
@decentralgabe this intro section has been reduced significantly and the analogy minimized considerably. I hope it addresses your concern!